tests/unit/ForgotPassword.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { shallowMount } from '@vue/test-utils'
+import ForgotPassword from '@/views/authentication/ForgotPassword.vue'
+import authApp from 'authenticationPlugin/App'
+const wrapper = shallowMount(ForgotPassword)
+beforeEach(() => {
+    wrapper.vm.$data.error = ""
+    wrapper.vm.$data.successMessage = ""
+    wrapper.vm.$data.email = ""
+})
+describe('ForgotPassword.vue', () => {
+    const _1stIndexOf_a = wrapper.findAll('a').at(0)
```

Comment:
bad idea. You should have id tags for each of these and you should be able to findById. if i change the structure of html all test would break which is a bad thing